### PR TITLE
feat: add basic bot documentation

### PR DESCRIPTION
**Pull request by proudlygeek**
## PR Description

This PR implements basic documentation for our new bot platform by refactoring the documentation in 2 main parts:

- `Bot Flow Builder`
- `Callbell API`

This will also allow to use the docs platform for future products.

### CODE REVIEW

**Review by Qwertic — APPROVED** [review]
Great job here 💯 it will be very helpful to have all the bits of the bot explained like this, specially the logic parts 👏🏼

File: docs/bot/getting_started.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+sidebar_position: 2
+---
+
+# Getting Started
+
+This step-to-step guid will walk you through building your first chatbot with Callbell.
```
Comment (camilamiz) [inline docs/bot/getting_started.md]:
```suggestion
This step by step guide will walk you through building your first chatbot with Callbell.
```

File: docs/bot/getting_started.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+## Adding a new action
+
+In order for the bot to work you'll have to add at least one action. For example, let's reply to the user with a message when he/she sends a message to the bot.
+
+Click on the **Add new action** button and select the **Send Message** action:
+
+![add_new_action](./assets/add_new_action.png)
+
+A panel will open you can use to configure the action. In this case we'll just add a simple text message:
```
Comment (camilamiz) [inline docs/bot/getting_started.md]:
```suggestion
A panel will open so you can use to configure the action. In this case we'll just add a simple text message:
```

File: docs/bot/getting_started.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+![run_simulator_reply](./assets/run_simulator_reply.png)
+
+## Publish your bot
+
+Once you're happy with your bot you can publish it by clicking on the **Publish Draft** button on the top right corner of the page:
+
+![publish_bot](./assets/publish_bot.png)
+
+After clicking on the **Publish** button and confirming your bot flow will be published; at the same time you'll need to **enable your bot** in order to make it available to your customers. You can do this by navigating back to the "Bots" section in the left sidebar and clicking on the **Enable bot** button:
```
Comment (camilamiz) [inline docs/bot/getting_started.md]:
```suggestion
After clicking on the **Publish draft** button and confirming, your bot flow will be published; at the same time you'll need to **enable your bot** in order to make it available to your customers. You can do this by navigating back to the "Bots" section in the left sidebar and clicking on the **Enable bot** button:
```

File: docs/bot/editor/actions/contact/add_tags.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+---
+sidebar_position: 2
+---
+
+import addTagsUrl from './assets/add_tags.png'
+
+# Add Tags
+
+<img src={addTagsUrl} width={180} />
```
Comment (camilamiz) [inline docs/bot/editor/actions/contact/add_tags.md:9]: Nice touch adding the icon! 💅

**Review by camilamiz — APPROVED** [review]
Great job! 📓 
Added some small suggestions.

**Pull request merged**